operator_ui/src/pages/Jobs/parseDot.ts

```diff
@@ -12,7 +12,14 @@ type Edge = {
 }
 
 export function parseDot(dot: string): Stratify[] {
-  const digraph = graphlibDot.read(dot)
+  // We want to permit the use of angle brackets to make the
+  // specs more readable for multi-line task attributes. The backend
+  // dot parsing library supports angle brackets which do not contain
+  // valid HTML inside them, but the frontend dot library graphlibDot does not.
+  // Since the dot parsing on the frontend is merely used to display the graph nodes
+  // its fine to omit the angle bracket attributes.
+  const dotNoAngleBrackets = dot.replace(/\w+=<.*>/g, '')
```

Comment:
```suggestion
  const dotNoAngleBrackets = dot.replace(/\w+\s*=\s*<.*>/g, '')
```

This should handle spacing between the key, equal sign and value.

For example
```
bla1231h = <"">
```